### PR TITLE
WAIV-3209, WAIV-2517, WAIV-2984, WAIV-3233, WAIV-3236

### PR DESCRIPTION
**Pull request by sergeiWizard**

### CODE REVIEW

File: src/client/components/EditorExtended/components/toolbar.less [hunk 1]
```diff
@@ -133,43 +123,43 @@
...
     }
   }
 }
 
 .md-RichEditor-controls {
   font-family: 'Helvetica', sans-serif;
-  font-size: 14px;
+  font-size: 13px;
   display: inline-block;
   border-right: 1px solid #555;
```
Comment (LucyHrinishyna) [inline src/client/components/EditorExtended/components/toolbar.less:143]: #555

File: src/client/components/EditorExtended/components/toolbar.less [hunk 2]
```diff
@@ -133,43 +123,43 @@
...
   position: relative;
 
-  &.md-RichEditor-show-link-input {
-    padding: 6px 10px;
-  }
+  //&.md-RichEditor-show-link-input {
+  //  padding: 6px 10px;
+  //}
```
Comment (LucyHrinishyna) [inline src/client/components/EditorExtended/components/toolbar.less:148]: комент

File: src/client/components/EditorExtended/components/toolbar.less (hunk 1 above)
Comment (sergeiWizard) [reply src/client/components/EditorExtended/components/toolbar.less]: fixed

File: src/client/components/EditorExtended/components/toolbar.less (hunk 2 above)
Comment (sergeiWizard) [reply src/client/components/EditorExtended/components/toolbar.less]: fixed

**Pull request merged**